.rubocop.yml

```diff
@@ -25,6 +25,7 @@ Metrics/BlockLength: { Exclude: [config/**/*.rb, spec/**/*.rb, db/seeds/**/*.rb]
 Metrics/MethodLength: { Exclude: [db/migrate/*.rb] }
 Rails/SkipsModelValidations: { AllowedMethods: [update_all] }
 RSpec/DescribeClass: { Exclude: [spec/system/**/*] }
+RSpec/ExampleLength: { Exclude: [spec/integration/feed_importing_spec.rb] }
```

Comment:
For exceptions like this, I would add them to `.rubocop_todo.yml` instead. I generally put permanent exceptions here, but the todo file is for stuff we hope to improve at some point.